pricing-page-exercise/merve/style.css

```diff
@@ -0,0 +1,277 @@
+}
+
+ul {
+    list-style-type: none;
+    width: 100%;
+}
+
+ul li {
+    display: inline;
+    width: 100%;
+    padding: 5px;
+
+}
+
+ul li :hover {
+    text-decoration-line: underline;
+}
+
+ul li a {
+    text-decoration: none;
+    color: black;
+    padding: 5px;
+}
+
+ul li hover {
+    text-decoration: dashed;
+}
+
+
+
+.pricing1 {
+    margin-top: 90px;
+    text-align: center;
+    width: 50%;
+    height: 100px;
+    margin-left: 25%;
+    margin-right: 25%;
+}
+
+.pricing {
+    font-size: 55px;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-weight: 200;
+    color: rgb(56, 56, 56);
+    line-height: 30px;
+}
+
+div p {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 50%;
+    height: 100px;
+    margin-top: -50px;
+    margin-left: 25%;
+    margin-right: 25%;
+    font-family: inherit;
+    color: rgb(104, 104, 104);
+    text-align: center;
+    font-size: 20px;
+    line-height: 30px;
+}
+
+.container {
+    width: 75%;
+    margin: 0 auto;
+    display: flex;
+    height: 330px;
+    justify-content: space-evenly;
+    margin-top: 45px;
+}
+
+.box {
+    text-align: center;
+    background-color: rgb(255, 255, 255);
+    width: 290px;
+    border: 1px solid rgba(149, 157, 165, 0.5);
+    border-radius: 5px;
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+}
+
+h2 {
+    text-align: center;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-size: 25px;
+    font-weight: normal;
+    color: rgb(48, 48, 48);
+    padding-bottom: 5px;
+    margin: 8px auto;
+}
+
+
+.line {
+    border-bottom: 1px solid rgba(168, 168, 168, 20%);
+
+}
+
+ol {
+    list-style-type: none;
+}
+
+ol li {
+    font-size: 15px;
+    margin: 10px;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    color: rgb(48, 48, 48);
+    line-height: 16px;
+}
+
+
+.dolar {
+    margin-top: -10px;
+    font-size: 40px;
+    margin-right: 10px;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
+.mo {
+    font-size: 35px;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    color: rgb(104, 104, 104)
+}
+
+.span1 {
+    margin-top: 20px;
+}
+
+.line2 {
+    border: 1px solid black;
+}
+
+.button2 {
+    border: 1px solid rgb(2, 136, 209);
+    padding: 10px;
+    color: rgb(2, 136, 209);
+    background-color: rgb(250, 252, 255);
+    text-align: center;
+    width: 85%;
```

Comment:
It's better to add padding to the "main" element, `.box` in this case to ensure that all elements inside of it are aligned identically. Then you can remove width from the button or set it to be 100%